rock_paper_scissors.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/python3
+print('''Let\'s play a game.
+''')
+def playgame():
+  player = input('Rock (1), paper (2) or scissors (3)?')
+  player = int(player)
+  if player ==1:
+    print ('You choose rock.')
+  elif player ==2:
+    print ('You choose paper.')
+  elif player ==3:
+    print ('You choose scissors.')
+  else:
+    print ('Please type 1, 2, or 3.')
+    playgame()
```

Comment:
Recursion is an interesting topic, but here I would look at loops to solve this problem (see my comment below on a potential structure for that). Interestingly, if you answer incorrectly enough times you'll get a stack overflow error and the program will crash.